go/vt/sqlparser/ast_rewriting_test.go

```diff
@@ -442,6 +426,27 @@ func TestRewritesSysVar(in *testing.T) {
 		in:       "select @x = @@sql_mode",
 		expected: "select :__vtudvx = :__vtsql_mode as `@x = @@sql_mode` from dual",
 		sysVar:   map[string]string{"sql_mode": "' '"},
+	}, {
+		in:       "SELECT @@tx_isolation",
+		expected: "select @@tx_isolation from dual",
+	}, {
+		in:       "SELECT @@transaction_isolation",
+		expected: "select @@transaction_isolation from dual",
+	}, {
+		in:       "SELECT @@session.transaction_isolation",
+		expected: "select @@session.transaction_isolation from dual",
+	}, {
+		in:       "SELECT @@tx_isolation",
+		expected: "select :__vttx_isolation as `@@tx_isolation` from dual",
+		sysVar:   map[string]string{"tx_isolation": "'READ-COMMITTED'"},
```

Comment:
nit: I found it difficult to read these cases, until I realised that sysVar is an input and not an output. `expected` should come last